domains/kayden.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "leekayden",
+        "email": "kaydenleefale@gmail.com"
+    },
+
+    "record": {
+        "A": ["a"]
```

Comment:
Please use a valid record.

Reply:
how can i add the correct record?

A @ 103.7.8.238
A www 103.7.8.238